device/rocev2/qps.yaml

```diff
@@ -0,0 +1,97 @@
+components:
+  schemas:
+    Rocev2.QPs:
+      description: >-
+            This allows the user to set QP properties between a particular source and destination.
+      type: object
+      required: [qp_name]
+      properties:
+        qp_name:
+          x-include: ../../common/common.yaml#/components/schemas/Named.Object/properties/name
+          x-field-uid: 1
+        connection_type:
+          $ref : '#/components/schemas/Rocev2.ConnectionType'
+          x-field-uid: 2
+    Rocev2.ConnectionType:
+      description: >-
+        It describes the degree of reliability and to what and how the QP transfers data. There are different types.
+      type: object
+      properties:
+        choice:
+          type: string
+          default: reliable_connection
+          x-enum:
+            reliable_connection:
+              x-field-uid: 1
+          x-field-uid: 1
+        reliable_connection:
+          $ref: '#/components/schemas/Rocev2.QPParameters'
+          x-field-uid: 2
+    Rocev2.QPParameters:
+      description: |-
+        ****Description here****
+      type: object
+      properties:
+        source_qp_number:
+          description: >-
+            Configure Source QP.
```

Comment:
The property name is "source_qp_number" but the description is "Configure Source QP". Name and description feel disconnected. Provide better text.